spec/spec.md

```diff
@@ -139,6 +139,19 @@ These features are deprecated, but can be supported by wallet endpoints to maint
 
 - [`solana:signTransactions`](#sign_transactions)
 
+## Account Profiles
```

Comment:
With this approach, I think it makes more sense to call these "Account Labels"

Reply:
I think account labels makes sense, adding context of when an account should be used. One potential one I see missing from this list is `identity` - an account that contains a users preferred identity credentials.